<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADD_data_material</class>
 <widget class="QDialog" name="ADD_data_material">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tạo vật liệu mới :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qml.qrc">
     <normaloff>:/dowload/Anh_cua_app+do_an/sign-add-icon.png</normaloff>:/dowload/Anh_cua_app+do_an/sign-add-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tên vật liệu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nhiệt Luyện</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kích Thước (S) :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Độ rắn :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Giới hạn bền uốn :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Giới hạn bền chảy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Công thức :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>102</y>
     <width>221</width>
     <height>196</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="Ten_vat_lieu"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>Toi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toi cai thien</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toi the tich</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tham Cacbon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tham Nito</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="kichthuoc"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="Do_ran"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>HB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HRC</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="Gioi_han_ben_u"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Gioi_han_ben_chay"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_3">
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtCurrent</enum>
      </property>
      <item>
       <property name="text">
        <string>2.8 x σch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40 x HRCm
</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
